firebase-config/ktx/src/test/kotlin/com/google/firebase/remoteconfig/ktx/RemoteConfigTests.kt

```diff
@@ -0,0 +1,130 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.remoteconfig.ktx
+
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.FirebaseApp
+import com.google.firebase.FirebaseOptions
+import com.google.firebase.remoteconfig.FirebaseRemoteConfig
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigValue
+import com.google.firebase.remoteconfig.FirebaseRemoteConfigBuilder
+import com.google.firebase.ktx.Firebase
+import com.google.firebase.remoteconfig.internal.ConfigCacheClient
+import com.google.firebase.remoteconfig.internal.ConfigFetchHandler
+import com.google.firebase.remoteconfig.internal.ConfigGetParameterHandler
+import com.google.firebase.remoteconfig.internal.ConfigMetadataClient
+import com.google.common.util.concurrent.MoreExecutors
+import com.google.firebase.ktx.app
+import com.google.firebase.ktx.initialize
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+
+const val APP_ID = "APP_ID"
+const val API_KEY = "API_KEY"
+
+const val EXISTING_APP = "existing"
+
+open class DefaultFirebaseRemoteConfigValue : FirebaseRemoteConfigValue {
+    override fun asLong(): Long = throw UnsupportedOperationException("Unimplementend")
```

Comment:
consider replacing throws with `TODO("Unimplemented")`

Reply:
Also, what is the motivation for this class, can't `StringRemoteConfigValue` directly extend `FirebaseRemoteConfigValue`?

Reply:
Updated to use `TODO("Unimplemented")`.

The main reason to have these 2 classes clarity. Having the boilerplate in the `DefaultFirebaseRemoteConfigValue` makes `StringRemoteConfigValue` trivial to understand.